articles/history-of-nodejs/history-of-nodejs.md

```diff
@@ -0,0 +1,167 @@
+    - [io.js](https://github.com/nodejs/iojs.org) became a major fork of Node.js "The Big Fork", with the purpose of introducing ES6 support and moving faster
+* **In 2015**
+    - The Node.js Foundation began
+    - IO.js is merged back into Node.js
+    - npm introduces private modules
+    - Node.js 4 (versions 1, 2 and 3 never previously released)
+* **In 2016**
+    - The leftpad incident
+    - The beginning of [Yarn](https://yarnpkg.com/)
+    - The beginning of Node.js 6
+* **In 2017**
+    - [npm]((https://www.npmjs.com/)) focuses more on security
+    - The beginning of Node.js 8
+    - HTTP/2
+    - [V8](https://v8.dev/) introduces Node.js in its testing suite, officially making Node.js a target for the JS engine, in addition to Chrome
+    - Up to3 billion npm downloads every week
+* **In 2018**
+    - The beginning of Node.js 10
+    - ES modules .mjs experimental support
+* **In 2019**
+    - The beginning of Node 12 - 13
+    - Work on [Deno](https://deno.land/) started to move server-side JS into the next decade with modern JavaScript support
+* **In 2020**
+    - The beginning of Node 14 - 15
+    - [GitHub](https://github.com/) (owned by Microsoft) acquired [NPM](https://www.npmjs.com/)
+
+### Setting up Node.js 	
+![Nodejs image setup](https://drive.google.com/uc?export=view&id=1zipo06_A5JDQiS3hfqO9TEiIjfwf5eua)
+[source](https://www.w3schools.com/nodejs/nodejs_get_started.asp)
+
+**First of all download Node.js**
+Installation instructions for Node.js can be viewed at the official Node.js website: https://nodejs.org
+
+After you have downloaded and installed Node.js on your computer, now we can try to display our first program "Hello World" in a web browser.
+
+Let's create our first Node.js file named _"firstfile.js"_, and add then the following code:
+
+> firstfile.js
+``` JavaScript
+const http = require('http');
+
+const requestListener = function (req, res) {
+  res.writeHead(200, {'Content-Type': 'text/html'});
+  res.end('Hello, World!');
+}
+
+const server = http.createServer(requestListener);
+server.listen(8080);
+```
+
+The file is saved on your computer: *C:\Users\YourName\firstfile.js*
+This code tells the computer to display "Hello World!" when anyone with a web browser tries to access your computer on port 8080.
+
+**Using Command Line Interface**
+To be able to access the Node.js files it must be initiated in the ["Command Line Interface" program of your computer.](https://www.w3schools.com/nodejs/nodejs_get_started.asp)
+Depending on your operating system, opening the command line interface on your computer would vary. If your are a Windows users, click on the start button and look for "Command Prompt", or you can simply write "cmd" in the search field for a quick search.
+
+After opening the 'cmd' go to the folder containing the file _"myfirstfile.js"_, the command line interface window should look something like this:
+
+`[C:\Users\YourName>](https://www.w3schools.com/nodejs/nodejs_get_started.asp)`
```

Comment:
I added this in code blocks - 
It had a > in front of it - but in .md file anything after > doesn't show up (or is added as a note) 
Not sure if this was intended.